raster-foundry/raster-foundry
# Remove Auth0 client secret configuration from repository

## Overview

Remove all references to the configuration required to thread through Auth0 client secret keys.

### Checklist


### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

After checking out this branch, pull down the latest environment variable configuration file:

```bash
$ ./scripts/bootstrap
```

From there, attempt to bring up the service and ensure that there are no errors due to missing Auth0 configuration:

```bash
$ ./scripts/server
```

Addresses half of https://github.com/azavea/raster-foundry-platform/issues/192#issuecomment-347580746
